### PR TITLE
Create database based on a diagram

### PR DESCRIPTION
**Pull request by ahmedadel56**
In this branch me and Alex trying to implement giving schema into a real database

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,42 @@
+CREATE TABLE patients (
+    id INT PRIMARY KEY GENERATED  ALWAYS AS IDENTITY,
+    name TEXT,
```
Comment (descholar-ceo) [inline schema_based_on_diagram.sql]: - [x] TEXT is too big, you can just optimize it by using `VARHCAR`

File: schema_based_on_diagram.sql [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    unit_price DECIMAL,
+    quantity INT,
+    total_price DECIMAL,
+    invoice_id INT,
+    treatment_id INT,
+    CONSTRAINT treatment_id FOREIGN KEY(treatment_id) REFERENCES treatments(id),
+    CONSTRAINT invoice_id FOREIGN KEY(invoice_id) REFERENCES invoices(id)
+
+);
+
```
Comment (descholar-ceo) [inline schema_based_on_diagram.sql:38]: - [x] all "join table" for the many-to-many relastionships should be created, even if they are not showed on the diagram. There fore you should create a table between medical_histories and treatments  (name can be different)

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
Hi @ahmedadel56 and @Wikitelmex , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Wikitelmex** [comment]
# changes required made
- change text to varchar(255)
- add auxiliary table to make the M-to-M relationship

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🎉🍾✔**
Hello @ahmedadel56 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**